### DISCUSSION TITLE
posts/google/

### DISCUSSION THREAD

**Discussion by giscus** [kpassapk/blog#3] [Q&A]
# posts/google/

I’ve had a sneaking suspicion for a whle that OKRs - as in Objectives and Key Results - are in fact clever device deployed by Google 1 to throw startups off track.
The appeal of OKRs is in their perceived simplicity: any problem large, or small, organizational or technical, made solvable by careful setting of O’s and KR’s.2
It never plays out as promised: a few cycles deep into OKRs will leave startups wondering why they’re so dumb they can’t implement what seems like a pretty simple system.

https://www.kpassa.me/posts/google/

**Comment by RicardoGeek** [comment]
I always enjoy a good rant about k8s :)

I miss the bare metal deployments era xD
